lib/auth/grpcserver.go

```diff
@@ -4478,8 +4479,8 @@ func (g *GRPCServer) GetInstaller(ctx context.Context, req *types.ResourceReques
 	if err != nil {
 		if trace.IsNotFound(err) {
 			switch req.Name {
-			case installers.InstallerScriptName:
-				return installers.DefaultInstaller, nil
+			case types.DefaultInstallerScriptName:
```

Comment:
why the change here and below to use `types.` instead of the `installers.` var? They are equal, so maybe we should delete one of them?

Reply:
There's no reason for changing things here, reverting this back to use `installers` package.

I'm not aware of the reason on why we have two constants for the same thing.